axelrod/tests/test_ecosystem.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for the Ecosystem class"""
+
+import unittest
+
+import axelrod
+
+
+class TestEcosystem(unittest.TestCase):
+
+    zeros = [[0 for i in range(10)] for i in range(10)]
+
+    def test_init(self):
+        """Are the populations created correctly?"""
+
+        eco = axelrod.Ecosystem(self.zeros)
+        self.assertEquals(eco.nplayers, 10)
+        self.assertEquals(len(eco.populations), 1)
+        self.assertEquals(len(eco.populations[0]), 10)
+        self.assertAlmostEqual(sum(eco.populations[0]), 1.0)
+
+    def test_reproduce(self):
+        """Does the reproduction mechanism work as expected?"""
+
+        eco = axelrod.Ecosystem(self.zeros)
+        eco.reproduce(100)
+        self.assertEquals(len(eco.populations), 101)
+        
```

Comment:
Could you please add a test for a full output (of course for small number of strategies and turns). This is just to ensure that changes further down the line don't break things.

Am I not at all convinced that `test_reproduce` does this comprehensively.